### ISSUE TITLE
Question About Finetune LLaMA

### ISSUE THREAD

**Issue by lxiaohaung** [expectorlin/NavCoT#3]
I ran the finetune code and the original weight was around 13g, but after finetune, it became almost 25g. Is this normal？

**Comment by expectorlin** [comment]
Yes. After finetuning, we set the "pretrained_path" parameter to the directories containing consolidated.*.pth weight files for inference.

**Comment by lxiaohaung** [comment]
Thank you very much for your patient reply. I have another question. I want to load the fine-tuned model to do some Q&A tests. Do you have this part of the code in your project? I can't seem to find it. It may also be that my coding ability is too poor.

**Comment by expectorlin** [comment]
The codes for Q&A tests are in LLaMA2-Accessory/accessory/demos. You can also refer to https://llama2-accessory.readthedocs.io/en/latest/inference.html for more details.

**Comment by lxiaohaung** [comment]
Thank you ！
